data/json/items/gun/9mm.json

```
@@ -716,8 +717,8 @@
     "looks_like": "hk_mp5",
     "type": "GUN",
     "reload_noise_volume": 10,
-    "name": { "str": "STEN" },
-    "description": "A British submachine gun developed during World War II, to make up for shortages of supplies.  Designed to be made using unskilled labor and not known for reliability, its main advantages were low cost and being able to use German 9mm ammunition.",
+    "name": { "str": "STEN MKII" },
+    "description": "The standard issue 9mm submachine gun of the British military throughout most of the Second World War, The STEN is aesthetically far from pretty.  Featuring an iconic side mounted magazine well and a design optimised for mass production using unskilled labour, the STEN was developed to make up for drastic supply shortages during the early days of the war.  Likely entering the US prior to the 1960s, either as a war time bringback or a museum purchase; you suppose that if you were to try and defend your nation, whatever the cost may be, and fight the undead ‘on the beaches, fight on the landing grounds, fight in the fields and in the streets, fight in the hills; and never surrender’, than it might be the time for the STEN to come back into frontline service.",
```

```suggestion
    "description": "The standard issue 9mm submachine gun of the British military throughout most of the Second World War. The STEN is aesthetically far from pretty.  Featuring an iconic side mounted magazine well and a design optimized for mass production using unskilled labor, the STEN was developed to make up for drastic supply shortages during the early days of the war.  Likely entering the US prior to the 1960s, either as a war time bringback or a museum purchase; you suppose that if you were to try and defend your nation, whatever the cost may be, and fight the undead ‘on the beaches, fight on the landing grounds, fight in the fields and in the streets, fight in the hills; and never surrender’, than it might be the time for the STEN to come back into frontline service.",
```
Technically neither word is a typo, but game set in America should probably use American spelling.

*Does not apply to DDotD 🍁